requirements_k8s.txt

```diff
@@ -1,7 +1,8 @@
 oauth2client
 kubernetes
+azure-cli==2.56.0
 google-api-python-client==2.96.0
-azure-mgmt-containerservice==22.0.0
-azure-identity==1.14.0
-azure-mgmt-resource==23.0.1
+azure-mgmt-containerservice==28.0.0
+azure-identity==1.14.1
+azure-mgmt-resource==23.1.0b2
```

Comment:
Same as above.

Reply:
I will unconditionally close this thread to keep discussions regarding this topic in single place